sdk/nodejs/connect.ts

```diff
@@ -6,8 +6,25 @@ import { Writable } from "node:stream"
  * ConnectOpts defines option used to connect to an engine.
  */
 export interface ConnectOpts {
+  /**
+   * Use to override Dagger workdir
+   */
   Workdir?: string
+  /**
+   * Use to override Dagger config
+   */
```

Comment:
Maybe add the default value in the doc in case those values are not overwritten

Reply:
Done 👍